packages/query/src/actions.ts

```diff
@@ -43,6 +43,7 @@ export async function startup(
       user: options.user,
       database: options.dbName,
       client_encoding: "'utf-8'",
+      application_name: 'pgtyped',
```

Comment:
> `application_name` (string)
>  The `application_name` can be any string of less than `NAMEDATALEN` characters (64 characters in a standard build). **It is typically set by an application upon connection to the server. The name will be displayed in the `pg_stat_activity` view and included in CSV log entries.** It can also be included in regular log entries via the log_line_prefix parameter. Only printable ASCII characters may be used in the application_name value. Other characters are replaced with C-style hexadecimal escapes.
> -- https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-APPLICATION-NAME

It's good practice to set this; but it also give people a way to `return null` rather than `raise exception` from their `STABLE` functions if it's `pgtyped` running.